taxiunn/client_auth/views.py

```diff
@@ -97,3 +103,66 @@ def post(self, request):
         except Exception as e:
             return Response({"error": str(e)},
                             status=status.HTTP_400_BAD_REQUEST)
+
+
+class PasswordRecoveryView(APIView):
+    def post(self, request):
+        serializer = PasswordRecoverySerializer(data=request.data)
+
+        if serializer.is_valid():
+            code = make_verification_code()
+            email = serializer.validated_data['email']
+            # запись данных в Redis
+            PasswordRecoveryCache.save(email=email, code=code)
+
+            # Отправка кода на почту
+            try:
+                send_verification_code(email=email,
+                                       verification_code=code)
+                return Response({'message': ('Check your email for '
+                                'the verification code.')},
+                                status=status.HTTP_200_OK)
```

Comment:
Бу испугался? не бойся я друг я тебя не обижу иди сюда иди ко мне сядь рядом со мной посмотри мне в глаза ты видишь меня я тоже тебя вижу давай смотреть друг на друга до тех пор пока наши глаза не устанут ты не хочешь почему что-то не так
```suggestion
                return Response(
                    {'message': (
                        'Check your email for the verification code.'
                    )},
                    status=status.HTTP_200_OK,
                )
```